src/ET_Client.php

```diff
@@ -629,7 +691,7 @@ function setRefreshToken($tenantKey, $refreshToken)
 	 * @param string $tenantKey
 	 * @return string Refresh token for the tenant
 	 */
-	function getRefreshToken($tenantKey) 
+	function getRefreshToken($tenantKey)
```

Comment:
please use "public function" instead of just "function" (not really an issue but for consistency)